render.yaml

```diff
@@ -6,12 +6,18 @@ services:
     repo: https://github.com/BenMemberg/how2meet.git
     region: ohio
     plan: free
-    branch: main
-    rootDir: how2meet
+    branch: tas-115_render_deploy
     buildCommand: poetry install
-    startCommand: poetry run python app.py
+    startCommand: poetry run make run_server
```

Comment:
So does `poetry run` here just tell it to run in a poetry shell effectively?

Reply:
yeah, that's basically how poetry works. everything after `run` happens within the poetry environment. Here its needed because `uvicorn` isn't installed directly in the server instance so the make command will fail